python/lancedb/remote/db.py

```diff
@@ -75,4 +76,21 @@ def create_table(
         on_bad_vectors: str = "error",
         fill_value: float = 0.0,
     ) -> Table:
+        if data is None and schema is None:
+            raise ValueError("Either data or schema must be provided.")
+        if data is not None:
+            data = _sanitize_data(
+                data, schema, on_bad_vectors=on_bad_vectors, fill_value=fill_value
+            )
+        else:
+            if schema is None:
+                raise ValueError("Either data or schema must be provided")
+            data = pa.Table.from_pylist([], schema=schema)
+
+        payload = {
+            "name": name,
+            "schema": schema_to_json(data.schema),
+            "records": data.to_pydict(),
+        }
+        self._loop.run_until_complete(self._client.create_table("/table/", payload))
```

Comment:
nit: I think this breaks a notebook. Don't know how to fix it yet. Jupyter will complain about loop already running.

Maybe using a lazy cache_property would fix the problem?

Reply:
Lazy property of the loop object?

Reply:
Nvm, just found the solution. https://stackoverflow.com/questions/46827007/runtimeerror-this-event-loop-is-already-running-in-python